ROADMAP.md

```diff
@@ -8,26 +8,16 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.14
+## Rook Ceph 1.15
 
-The following high level features are targeted for Rook v1.14 (April 2024). For more detailed project tracking see the [v1.14 board](https://github.com/rook/rook/projects/31).
+The following high level features are targeted for Rook v1.15 (July 2024). For more detailed project tracking see the [v1.15 board](https://github.com/rook/rook/projects/32).
 
-* Support for Ceph Squid (v19)
-* Allow setting the application name on a CephBlockPool [#13744](https://github.com/rook/rook/pull/13744)
-* Pool sharing for multiple object stores [#11411](https://github.com/rook/rook/issues/11411)
-* DNS subdomain style access to RGW buckets [#4780](https://github.com/rook/rook/issues/4780)
 * Replace a single OSD when a metadataDevice is configured with multiple OSDs [#13240](https://github.com/rook/rook/issues/13240)
```

Comment:
1) Support logrotate for csi pods https://github.com/rook/rook/issues/12809
2) Support Automatically decide type of MDS pod scaling  https://github.com/rook/rook/issues/14185
3) For external cluster make import and export more easier https://github.com/rook/rook/issues/14077
4) Support key rotation for cephobjectstore user https://github.com/rook/rook/issues/11563

Reply:
I'll add the first and fourth. The second and third don't feel defined enough yet to add them to this roadmap doc.